### PR TITLE
Add help to add annotation and network header modal dialog

### PR DESCRIPTION
**Pull request by p3rcypj**
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/1xwfh93

### :memo: Implementation

Added help icon with tooltip on hover in "add annotation" and "network" header modal dialog.

### :art: Screenshots

image
image

### CODE REVIEW

File: app/assets/javascripts/3dbio_viewer/src/webapp/components/HtmlTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Theme, Typography, withStyles } from "@material-ui/core";
+import { Tooltip } from "@material-ui/core";
+import styled from "styled-components";
+
+export const HtmlTooltip = withStyles((theme: Theme) => ({
```
Comment (tokland) [inline app/assets/javascripts/3dbio_viewer/src/webapp/components/HtmlTooltip.tsx:5]: [nothing do do] HOC-styling (Higher Order Components, in general) has gone out of fashion, but I see this is coming from the covid19 app, so let's just leave it.

File: app/assets/javascripts/3dbio_viewer/src/webapp/components/annotations-tool/AnnotationsTool.tsx [hunk 2]
```diff
@@ -99,12 +101,25 @@ export const AnnotationsTool: React.FC<AnnotationsToolProps> = React.memo(props
 
     return (
         <Dialog open={true} onClose={onClose} maxWidth="lg">
-            <DialogTitle>
+            <StyledDialogTitle>
                 {i18n.t("Add annotation")}
+                <HtmlTooltip
```
Comment (tokland) [inline app/assets/javascripts/3dbio_viewer/src/webapp/components/annotations-tool/AnnotationsTool.tsx:106]: This pattern is duplicated in Network.tsx, maybe can we create a component `<DialogHelp><StyledTypography variant="body2">...</DialogHelp>`?

**Comment by p3rcypj** [comment]
Thanks for your review @tokland!

**Review by tokland — APPROVED** [review]
:+1: All looking good! Thanks, @p3rcypj

**Pull request merged**